x-pack/solutions/security/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_finding_flyout.tsx

```diff
@@ -5,390 +5,46 @@
-    <EuiFlexItem grow={false}>
-      <CopyButton copyText={item} />
-    </EuiFlexItem>
-  </EuiFlexGroup>
-);
-
-const getFlyoutDescriptionList = (
-  vulnerabilityRecord: CspVulnerabilityFinding
-): EuiDescriptionListProps['listItems'] =>
-  [
-    vulnerabilityRecord.resource?.id && {
-      title: i18n.translate(
-        'xpack.csp.vulnerabilities.vulnerabilitiesFindingFlyout.flyoutDescriptionList.resourceId',
-        { defaultMessage: 'Resource ID' }
-      ),
-      description: vulnerabilityRecord.resource.id,
-    },
-    vulnerabilityRecord.resource?.name && {
-      title: i18n.translate(
-        'xpack.csp.vulnerabilities.vulnerabilitiesFindingFlyout.flyoutDescriptionList.resourceName',
-        { defaultMessage: 'Resource Name' }
-      ),
-      description: vulnerabilityRecord.resource.name,
-    },
-    vulnerabilityRecord.package?.name && {
-      title: i18n.translate(
-        'xpack.csp.vulnerabilities.vulnerabilitiesFindingFlyout.flyoutDescriptionList.packageTitle',
-        { defaultMessage: 'Package' }
-      ),
-      description: Array.isArray(vulnerabilityRecord.package?.name) ? (
-        <div
-          css={css`
-            display: inline-block;
-          `}
-        >
-          <MultiValueCellPopover<CspVulnerabilityFinding>
-            items={vulnerabilityRecord.package?.name}
-            field={'package.name'}
-            object={vulnerabilityRecord}
-            renderItem={renderFinding}
-          />
-        </div>
-      ) : (
-        vulnerabilityRecord.package?.name
-      ),
-    },
-    vulnerabilityRecord.package?.version && {
-      title: i18n.translate(
-        'xpack.csp.vulnerabilities.vulnerabilitiesFindingFlyout.flyoutDescriptionList.versionTitle',
-        { defaultMessage: 'Version' }
-      ),
-      description: Array.isArray(vulnerabilityRecord.package?.version) ? (
-        <div
-          css={css`
-            display: inline-block;
-          `}
-        >
-          <MultiValueCellPopover<CspVulnerabilityFinding>
-            items={vulnerabilityRecord.package?.version}
-            field={'package.version'}
-            object={vulnerabilityRecord}
-            renderItem={renderFinding}
-          />
-        </div>
-      ) : (
-        vulnerabilityRecord.package?.version
-      ),
-    },
-  ].filter(truthy);
-
-export const VulnerabilityFindingFlyout = ({
-  closeFlyout,
-  vulnerabilityRecord,
-  onPaginate,
-  totalVulnerabilitiesCount,
-  flyoutIndex,
-  isLoading = false,
+const VulnerabilityFindingFlyout = ({
+  vulnerabilityId,
+  resourceId,
+  packageName,
+  packageVersion,
+  eventId,
+  children,
 }: {
-  closeFlyout: () => void;
-  vulnerabilityRecord: CspVulnerabilityFinding;
-  onPaginate?: (pageIndex: number) => void;
-  totalVulnerabilitiesCount?: number;
-  flyoutIndex?: number;
-  isLoading?: boolean;
+  vulnerabilityId: string | string[];
+  resourceId: string;
+  packageName: string | string[];
+  packageVersion: string | string[];
```

Comment:
why is the type `string | string[]` for the package version and vulnerability id?

Reply:
because it is possible for it to have multiple value if it came from 3rd party